build/devServer.js

```diff
@@ -0,0 +1,33 @@
+const port = process.env.MAPSTORE_BACKEND_PORT || 8080;
```

Comment:
Note: we have a similar concept of devServer inside the [mapstore-project](https://github.com/geosolutions-it/mapstore-project/blob/master/types/standard/config/index.js#L173) but in that case it return a function.
Here an example of use case https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/devServer.js

Reply:
Interesting, it can be generalized for projects. In this version I allowed also to add `devtool` option. (to use `source-maps` or other settings.